Magento/Sniffs/PHP/ReturnValueCheckSniff.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Sniffs\PHP;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Detects misusing of IS_IDENTICAL operators.
```

Comment:
What are we trying to detect with this sniff, briefly looked at `Magento/Tests/PHP/ReturnValueCheckUnitTest.inc` but cannot get the purpose.

Reply:
Strict comparison should be user to check the return value for `strpos`, `stripos`,
`array_search`, because they can return boolean `false`, and may also return a non-boolean value which evaluates to `false`.

http://php.net/manual/en/function.strpos.php